CONTRIBUTING.md

```diff
@@ -0,0 +1,57 @@
+This document is a guide for contributions to the WGPU project.
+
+## Welcome!
+
+First of all, welcome to the WGPU community! 👋 We're glad you want to
+contribute. If you are unfamiliar with the WGPU community as an organization, we
+recommend you read [`GOVERNANCE.md`] for an overview of how it works.
+
+[`GOVERNANCE.md`]: ./GOVERNANCE.md
+
+## Talking to other humans in the WGPU project
+
+The WGPU project has multiple ways to engage:
+
+- [GitHub discussions]: TODO: not sure when to use this over Matrix and issues.
+  TBH.
+- The Matrix channel [`wgpu:matrix.org`](https://matrix.to/#/#wgpu:matrix.org)
+  is dedicated to informal chat about contributions the project. It is
+  particularly useful for:
+  - …saying hello, and introducing yourself.
+  - …validating contributions (i.e., determining if they'll be accepted,
+    ensuring your approach is correct, making sure you aren't wasting effort,
+    etc.)
+  - …setting expectations for contributions 
+
+  Notification in Matrix can sometimes be unreliable. Feel free to explicitly tag
+  people from whom you would like attention (but don't abuse it!), and follow-up
+  after a day or so if you do not get a response to your messages.
+
+- [GitHub issues] are for action items that the community needs to prioritize.
+  These could be items of work resolved by pull requests, decisions to make with
+  a formal outcome, or collections of issues needed for a specific feature or
+  use case (AKA `[meta]` issues).
+- [GitHub pull requests]: TODO
```

Comment:
todo

Reply:
"Almost all modifications of this repository happen via pull requests including for maintainers of the project."

I'd remove the "almost" but someone will find some edge case and prove me wrong. I don't think that we need to formalize what a pull request is beyond beyond that.